docs/mkdocs/docs/tutorials/data_organisation.md

```diff
@@ -0,0 +1,233 @@
+
+* Ideal for single security analysis
+* The delivery format for the data is often per security
+* The update process is simple
+
+#### Cons
+
+* Analysis involving many securities requires reading many symbols
+* Time index for raw data may not match between securities.
+
+The data is easier to use with [time index alignment](#time-index-alignment).
+
+
+### Single Symbol for all Securities
+
+The data for all securities is merged together into a single symbol.
+
+![](../images/MarketData5MinBarsAllSecs.png)
+
+The security identifier is included as an extra index column. Note that DataFrames with a Pandas MultiIndex will round trip correctly, which is useful. However the high performance ArcticDB indexing is only on the primary index level.
+
+#### Pros
+
+* Good for analysis involving many securities at once eg. portfolio analysis
+* A single large DataFrame tends to give very good read performance
+
+#### Cons
+
+* The update process is more complicated. Probably needs a read, modify, write sequence of operations (although see [Future Features](#planned-future-features-and-improvements))
+* Time index for raw data may not match between securities.
+
+The data is easier to use with [time index alignment](#time-index-alignment).
+
+
+### Single Symbol for each Data Item
+
+A symbol holds the timeseries for all securities for a single field. In the example below are the close prices over time for all securities.
+
+![](../images/MarketData5MinBarsAllSecsClose.png)
+
+#### Pros
+
+* Good for analysis involving many securities at once eg. portfolio analysis
+* A single large DataFrame tends to give very good read performance
+* Update process is reasonably simple.
+
+#### Cons
+
+* Time index for raw data may not match between securities. 
+
+The data is easier to use with [time index alignment](#time-index-alignment).
+
+
+
+## Time Index Alignment
+
+Raw data may not be time-aligned
+
+* The underlying tick data arrives at arbitrary times
+* Different products have different trading hours
+* Some securities may trade less frequently
+
+For many purposes time-aligned data is either necessary or makes the task much simpler.
+
+Resampling is one way to align data. There is a Pandas resample function and a faster function planned in ArcticDB (see [Future Features](#planned-future-features-and-improvements)).
+
+
+## Enterprise Features
+
+Many of the housekeeping and maintenance procedures recommended in this guide are taken care of by processes available in the ArticDB Enterprise package. Please contact us if you would like to explore further.
+
+
+## Planned Future Features and Improvements
+
+We are always improving and adding new features to ArcticDB. Below are a few relevant new features that are in our plans
+
+* **Multi-Index:** support for more flexible data updates for multi-index symbols.
+* **Resampling:** a planned [QueryBuilder](../api/query_builder) extension that will allow fast on-the-fly resampling of data as part of the read.
+

```

Comment:
do we have a process for users to request features? maybe raising an issue / feature request?